block.go

```diff
@@ -89,6 +89,7 @@ type Disk struct {
 	SizeBytes              uint64            `json:"size_bytes"`
 	PhysicalBlockSizeBytes uint64            `json:"physical_block_size_bytes"`
 	DriveType              DriveType         `json:"drive_type"`
+	Removable              bool              `json:"removable"`
```

Comment:
How about naming this `IsRemovable` to match the `IsReadOnly` boolean attribute of `Partition`?